### ISSUE TITLE
Failed: FtpsStageTest.move

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#1371]
```
[error] Test akka.stream.alpakka.ftp.FtpsStageTest.move failed: java.lang.AssertionError: expected:<IOResult(1,Success(Done))> but was:<IOResult(1,Failure(javax.net.ssl.SSLException: bad record MAC))>, took 0.182 sec
[error]     at akka.stream.alpakka.ftp.CommonFtpStageTest.move(CommonFtpStageTest.java:129)
```

https://travis-ci.org/akka/alpakka/jobs/467503694#L995

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/475052674

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/486971687

**Comment by tgambet** [comment]
I believe this can be safely closed since #1668.

**Comment by ennru** [comment]
Thank you.
Expected to be solved with #1668.
